tasks/html-css-popup.md

```diff
@@ -25,6 +25,40 @@ The goal of this task is to practice coding HTML & CSS from mockup. You are expe
     makes the content of the popup scrollable
 - mobile view is irrelevant for this task
 
+## Recommendations
+- Make sure that your navigation is built by this structure: 
+``` HTML
+  <nav>
+    <ul>
+      <li>
+        <a href='#'>Link</a>
+      </li>
+    <ul>
+  </nav>
+```
+- Make sure that you've used visually-hidden for hiding inputs:
+``` CSS
+.visually-hidden {
```

Comment:
Class `.visually-hidden` used only if you need an IE11 support? for modern browsers they can use 
```
appearance: none;
-moz-appearance: none;
-webkit-appearance: none;
```

Reply:
Ok, thanks! I'll add these styles.

Reply:
@A-Ostrovnyy, updated

Reply:
As for me better to add a link on [video](https://www.youtube.com/watch?v=E6kLaaQFctU&ab_channel=VadimMakeev) instead of a production-ready solution.

Reply:
Agree